files/fr/web/css/aspect-ratio/index.html

```diff
@@ -0,0 +1,70 @@
+---
+title: aspect-ratio
+slug: Web/CSS/aspect-ratio
+browser-compat: css.properties.aspect-ratio
+translation_of: 'Web/CSS/aspect-ratio'
+---
+<div>{{CSSRef}}</div>
+
+<p>La propriété <a href="/fr/docs/Web/CSS">CSS</a> <strong><code>aspect-ratio</code></strong> définit un <strong>ratio d'affichage préférentiel</strong> pour la boîte, qui sera utilisé dans le calcul des tailles automatiques et pour d'autres fonctions de mise en page.</p>
+
+<pre class="brush:css no-line-numbers">aspect-ratio: 1 / 1;
+
+/* Valeurs globales */
+aspect-ratio: inherit;
+aspect-ratio: initial;
+aspect-ratio: revert;
+aspect-ratio: unset;
+</pre>
+
+<h3 id="values">Valeurs</h3>
+
+<dl>
+ <dt>{{cssxref("&lt;auto&gt;")}}</dt>
+ <dd><a href="/en-US/docs/Web/CSS/Replaced_element">Replaced elements</a> with an intrinsic aspect ratio use <em>that</em> aspect ratio, otherwise the box has no preferred aspect ratio. Size calculations involving intrinsic aspect ratio always work with the content box dimensions.</dd>
+ <dt>{{cssxref("&lt;ratio&gt;")}}</dt>
+ <dd>The box’s preferred aspect ratio is the specified ratio of <code>width</code> / <code>height</code>. Size calculations involving preferred aspect ratio work with the dimensions of the box specified by <code>box-sizing</code>.</dd>
+</dl>
+
+<h2 id="formal_definition">Définition formelle</h2>
+
+<p>{{cssinfo}}</p>
+
+<h2 id="formal_syntax">Syntaxe formelle</h2>
+
+<p>{{csssyntax}}</p>
+
+<h2 id="examples">Exemples</h2>
+
+<h3 id="examples_of_values_for_aspect-ratio">Exemples de valeurs pour aspect-ratio</h3>
+
+<pre class="brush: css">aspect-ratio: 1 / 1;
+aspect-ratio: 16 / 9;
+</pre>
+
+<h2 id="mapping_width_and_height_to_aspect-ratio">Correspondance entre largeur et hauteur avec aspect-ratio</h2>
+
+<p>Les navigateurs ont ajouté une propriété <code>aspect-ratio</code> interne qui s'applique aux <a href="/fr/docs/Web/CSS/Replaced_element">éléments remplacés</a> et à d'autres éléments liés qui acceptent les attributs <code>width</code> et <code>height</code>. Cela se passe dans la feuille de styles interne UA du navigateur.</p>
```

Comment:
It seems to me that "feuille de styles interne UA du navigateur" is a bit superfluous/complex vs "feuille de styles interne du navigateur" (as the User Agent is the browser anyway).
If there are various internal stylesheets, please discard my comments.

Reply:
LGTM

Reply:
Fixed in https://github.com/mdn/translated-content/pull/1722/commits/91db71aaecc43376852a5c0c93fb01acaabcd47c